### DISCUSSION TITLE
Alternative / Spiritual Successor https://github.com/werman/noise-suppression-for-voice

### DISCUSSION THREAD

**Discussion by onlyreportingissues** [noisetorch/NoiseTorch#267] [General]
https://github.com/werman/noise-suppression-for-voice

**Comment by CheaterTim** [comment]
I am using [cadmus](https://github.com/josh-richardson/cadmus) as alternative for now. its easy to use with a nice gui

**Comment by lawl** [comment]
how can it be a spiritual successor if i contributed the algorithm people seem to like about noisetorch there before actually making noisetorch :thinking: ?
